sources/libs/ipc/hooks.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <ipc/component.h>
+
+typedef enum
+{
+    IDL_HANDLER_REQ_RESP,
+    IDL_HANDLER_NIL_RESP,
+    IDL_HANDLER_REQ_NIL,
+    IDL_HANDLER_NIL_NIL,
```

Comment:
why not put NIL_NIL to the top so it is 0 and not 3 ?

Reply:
Fix